#----------------------
# Point colors
#----------------------
int32 COLOR_BLUE=0
int32 COLOR_GREEN=1
int32 COLOR_RED=2
int32 COLOR_YELLOW=3
int32 COLOR_OTHER=4

#----------------------
# Point info
#----------------------
int32 color
int32 x
int32 y
